tools/search_files.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# Copyright 2008 the V8 project authors.
+# Copyright 2023 Microsoft Inc.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import glob
+import os
+import sys
+
+def SearchFiles(root, dir, ext):
+  list = glob.glob(dir + '/**/*.' + ext, recursive=True)
+  if sys.platform == 'win32':
+    list = [ x.replace('\\', '/') for x in list]
+  list = [ os.path.relpath(x, root) for x in list ]
+  return sorted(list)
```

Comment:
This is almost the same as the existing
https://github.com/nodejs/node/blob/de5787c87c557e02523b3e6104445c6427a44f25/tools/utils.py#L113-L117